src/Files.App/Data/Commands/HotKey/HotKeyCollection.cs

```diff
@@ -23,7 +23,7 @@ namespace Files.App.Data.Commands
 		public int Length => hotKeys.Length;
 
 		public string Code => string.Join(',', hotKeys.Select(hotKey => hotKey.Code));
-		public string Label => string.Join(',', hotKeys.Where(hotKey => hotKey.IsVisible).Select(hotKey => hotKey.Label));
+		public string Label => string.Join(", ", hotKeys.Where(hotKey => hotKey.IsVisible).Select(hotKey => hotKey.Label));
```

Comment:
Do you think one space is enough?

Reply:
> Do you think one space is enough?

Two spaces looked too wide for me personally. But if you want two spaces, I can spend a couple hours to make it happen. :P

Reply:
I'll take your word, once space it is 👍

Reply:
Actually, now that I look at it on a 4k monitor instead of 1080, two spaces looks somewhat more necessary.